main_test.go

```diff
@@ -39,3 +39,10 @@ func TestYAMLParse(t *testing.T) {
 	assert.Contains(t, java.Environment, "ELASTIC_APM_LOG_LEVEL")
 	assert.Contains(t, java.Environment, "ELASTIC_APM_PROFILING_INFERRED_SPANS_ENABLED")
 }
+
+func TestYAMLParseBad(t *testing.T) {
+	p := "testdata/bad.yaml"
```

Comment:
Should we also test bad2.yaml? It doesn't appear to be used?

Reply:
Good catch!